libcontainer/configs/mount_linux.go

```diff
@@ -29,8 +31,21 @@ type Mount struct {
 
 	// Extensions are additional flags that are specific to runc.
 	Extensions int `json:"extensions"`
+
+	// UID/GID mappings used for changing file owners w/o calling chown, fs should support it.
+	// Every mount point could have its own mapping.
+	UIDMappings []IDMap `json:"uidMappings"`
+	GIDMappings []IDMap `json:"gidMappings"`
 }
 
 func (m *Mount) IsBind() bool {
 	return m.Flags&unix.MS_BIND != 0
 }
+
+func (m *Mount) IsBindOnly() bool {
```

Comment:
```suggestion
func (m *Mount) IsBindWithoutIDMap() bool {
```

Reply:
@AkihiroSuda I'm not convinced that is better. It is all pretty cosmetic, so I'll commit your suggestion, but in this matter of personal taste, I'd say I prefer the current name. If tomorrow we have another kind of mount (bind mounts that are XXX too, besides bind mounts that are idmap) then we would want here to be `bindWithoutIDMapWithoutXX()`?

It seems not nice IMHO. While if this is _only_ a bind mount is what we want (not a bindmount that is also idmap, or XXX, etc.).

What do you think?

Reply:
As we discussed in other comments, I've just removed this and open-code it whenever I need it.  Thanks!